addon/services/cookies.js

```diff
@@ -5,8 +5,9 @@ import { assert } from '@ember/debug';
 import { A } from '@ember/array';
 import { getOwner } from '@ember/application';
 import Service from '@ember/service';
-import { merge } from '@ember/polyfills';
-const { keys, assign } = Object;
+import { merge, assign as emberAssign } from '@ember/polyfills';
+const { keys } = Object;
+const assign = Object.assign || emberAssign || merge;
```

Comment:
I guess `emberAssign` would always be present so the fallback to `|| merge` is not necessary? We could even just always use `emberAssign`.

Reply:
Ember.assign was added in Ember 2.5, so the merge fallback is there for older version support. It's a fairly common pattern, I've seen.

Reply:
Yes, you're right of course 🤦‍♂️

Reply:
Heh, no worries -- I do that all the time.

Thanks for the merge! :)